### DISCUSSION TITLE
how do i import previously installed game

### DISCUSSION THREAD

**Discussion by shafiahaz2478** [legendary-gl/legendary#221] [General]
Hi,
how do I import previously installed game  I don't understand the instruction for it on the readme.md
I am trying to import rocket league

edit: I figured it out instead of typing rocket league just type sugar don't know why

**Comment by user4302** [comment]
thats because thats the keyword used by Epic Games to identify Rocket League.
you can view all game keyword names by running legendary list-games (will list games that you own)

**Reply by shafiahaz2478** [reply]
i knew on that

**Comment by JF224** [comment]
You can run `legendary -y egl-sync` and it will sync all games fromt he Epic Games Launcher.
